data/FilterList.json

```diff
@@ -7,7 +7,6 @@
     "name": "280 Blocker",
     "submissionUrl": "https://docs.google.com/forms/d/e/1FAIpQLScNeZhFrFZt9GhIVGdThGz7oyepcNRKuOi5PJDnsC-awxTeOQ/viewform",
     "syntaxId": 3,
-    "updatedDate": "2018-10-10T08:45:09",
```

Comment:
Was there a reason you removed this date? It's an auto-updated field when FilterLists senses a list is updated, but can also be specified manually for lists that haven't been updated in a long time (pre-FilterList's scans). Let's keep the date in for now unless you have a reason to remove it. [More Info](https://github.com/collinbarrett/FilterLists/wiki/FilterList)